seed/lib/superperms/orgs/models.py

```diff
@@ -140,14 +139,16 @@ def save(self, *args, **kwargs):
         # Create a default cycle for the organization if there isn't one already
         from seed.models import Cycle
         year = date.today().year - 1
-        cycle_name = 'Default ' + str(year) + ' Calendar Year'
+        cycle_name = '{} Calendar Year'.format(year)
         if not Cycle.objects.filter(name=cycle_name, organization=self).exists():
```

Comment:
I was noticing that I was ending up with 2 default cycles anytime I was created an new organization, one from this point in the code and one from seed/management/commands/migrate_organization.py
Both places are checking for an existing cycle for the calendar year based on cycle name, but one was using "{year} Calendar Year" and one was using "Default {year} Calendar Year".  Since one of the two cycle names needed to be changed to insure only one calendar year is created per org creation, removing the "Default" from this calendar year name seemed the way to go since it does also make it cleaner, and brings it in line with an expedited filtering by calendar year through filtersets.